### ISSUE TITLE
[UltralyticsHub] pretrained_model seems not working during training session

### ISSUE THREAD

**Issue by siegkeron** [snuailab/waffle_hub#159]
### Search before asking

- [X] I have searched the [issues](https://github.com///issues) and found no similar bug report.

### Bug

I tested two training option(w/ or w/o "pretrained_model")with same dataset.

they shows almost same training results.

In my opinion, using "pretrained_model" shows lower train loss and faster converge.

Training configs and results below.

### w/ "pretrained_model" 
```
train_callback = hub.train(
    dataset_path=dataset_dir,
    epochs=100,
    batch_size=16,
    image_size=[640,480],
    learning_rate=0.003,
    letter_box=None,
    pretrained_model="hubs/model_scene_cls_round5/weights/best_ckpt.pt",
    device="0",
    workers=2,
    seed=0,
    verbose=True,
    hold=True
)
```
![results_r6](https://github.com/snuailab/waffle_hub/assets/90073692/376c6d55-7fae-4652-ae28-9d4a3ffd8111)

### w/o  "pretrained_model" 
```
train_callback = hub.train(
    dataset_path=dataset_dir,
    epochs=100,
    batch_size=16,
    image_size=[640,480],
    learning_rate=0.003,
    letter_box=None,
    pretrained_model=None,
    device="0",
    workers=2,
    seed=0,
    verbose=True,
    hold=True
)
```
![results_rand_init](https://github.com/snuailab/waffle_hub/assets/90073692/e1ef1886-dda5-472f-ab83-6f22cd25d72b)

**Comment by ZeroAct** [comment]
@siegkeron 
It seems akward.
I'll let you know after debugging.

**Comment by dustjr1565** [comment]
I attempted to reproduce it, but there are no anomalies in the results as seen in the logs.

1. default_yolo_weight
```
epoch : AP50-95
1: 0.08604
2: 0.06758
...
```
2. use pretrained-model
```
epoch : AP50-95
1: 0.96781
2: 0.96795
...
```

**Comment by ZeroAct** [comment]
@dustjr1565 
Did you check dlt hub?
It seems that @siegkeron  used dlt hub.

**Comment by dustjr1565** [comment]
>  refer to the issue title.

**Comment by dustjr1565** [comment]
> @dustjr1565 Did you check dlt hub? I'm not sure, but it seems that @siegkeron used dlt hub.
> 
> Am i right?

 refer to the issue title.
